src/module/migration/migrations/784-comp-browser-pack-setting.ts

```diff
@@ -0,0 +1,14 @@
+import { MigrationBase } from "../base";
+
+/** Migrate Compendium Browser packs selection setting to an object */
+export class Migration784CompBrowserPackSetting extends MigrationBase {
+    static override version = 0.784;
+
+    override async migrate() {
+        const savedSettings = game.settings.get("pf2e", "compendiumBrowserPacks");
+        if (!savedSettings.action) {
```

Comment:
This is what the setting looks like in master:
![image](https://user-images.githubusercontent.com/41452412/189399329-ae9b5231-dcc5-45e2-a64d-27ebc180799c.png)
It would probably be save to omit the check completely as the migration should only hit old settings.

Reply:
This behavior has changed in the last couple versions due to https://github.com/foundryvtt/foundryvtt/issues/7839

It's an easy check to make and will protect against worlds in different states.

Reply:
The problem is that the returned value is in fact not a string.
![image](https://user-images.githubusercontent.com/41452412/189409348-cc0cde80-fcc7-4f55-8b21-c8bebfd2cc6d.png)

Reply:
Looks like `instanceof String` works. I'll update it,